src/classes/STG_SettingsManager_CTRL.cls

```diff
@@ -129,6 +134,14 @@ public with sharing class STG_SettingsManager_CTRL extends STG_Panel {
             if (!Test.isRunningTest()) {
                 upsert STG_Panel.stgService.stgAllo;
             }
+            if (STG_Panel.stgService.stgAllo.Payment_Allocations_Enabled__c) {
+                ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.ERROR, Label.stgGauWithPaymentEnabledError));
+                ERR_Handler.processError(new HealthCheckException(Label.stgGauWithPaymentEnabledError), ERR_Handler_API.Context.STTG);
+            }
+            else {
+                ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.ERROR, Label.stgGauWithOutPaymentEnabledError));
+                ERR_Handler.processError(new HealthCheckException(Label.stgGauWithOutPaymentEnabledError), ERR_Handler_API.Context.STTG);
+            }
```

Comment:
There are two syntax issues with the above. The first is the `else` on a line by itself, and the other is that we standardize on using `System.Label...`. Though, I think you can rewrite this if block above as:

```suggestion
           String errorMsg;
           if (STG_Panel.stgService.stgAllo.Payment_Allocations_Enabled__c == true) {
                 errorMsg = System.Label.stgGauWithPaymentEnabledError;
            } else {
                 errorMsg = System.Label.stgGauWithOutPaymentEnabledError;
            }
            ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.ERROR, errorMsg));
            ERR_Handler.processError(new HealthCheckException(errorMsg), ERR_Handler_API.Context.STTG);
```

Reply:
The code review changes are all done and ready for review. @force2b , I added your suggestions for extra white spaces. I noticed it updated automatically some comments in the class.